google/data_source_google_compute_region_instance_group_manager.go

```diff
@@ -0,0 +1,161 @@
+package google
+
+import (
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	compute "google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+	"log"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+func dataSourceGoogleComputeRegionInstanceGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceComputeRegionInstanceGroupRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"items": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"instance": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"status": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"named_ports": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"name": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"port": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"project": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"self_link": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"size": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceComputeRegionInstanceGroupRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	var project, region, name string
+	// add self_link parsing here.
```

Comment:
It looks like you've already added self_link parsing below

Reply:
Ha, true, removed.